lib/circlex/api/payments/payments.ex

```diff
@@ -102,21 +102,24 @@ defmodule Circlex.Api.Payments.Payments do
 
       iex> host = Circlex.Test.start_server()
       iex> amount = %{amount: "12345.00", currency: "USD"}
-      iex> Circlex.Api.Payments.Payments.mock_wire("CIR3KX3L99", amount, host: host)
+      iex> beneficiaryBank = %{accountNumber: "1000000001"}
```

Comment:
We should keep every variable in Elixir as `snake_case`. Also, why not just have the user pass in an `account_number` and not some map?